convert2rhel/backup/__init__.py

```diff
@@ -48,6 +48,7 @@ class BackupController:
 
     def __init__(self):
         self._restorables = []
+        self._rollback_failed = False
```

Comment:
@hosekadam, do you mind updating the class docstring to include those two properties? 

Just know I realized that we don't have a description even for `_restorables`, so it would be good to update it and include both of them.

Reply:
I will do that. Just I'll wait a bit till the PR will be close to merging